src/hw/actions/app.ts

```diff
@@ -634,7 +634,8 @@ export const createAction = (
       ...state,
       inWrongDeviceForAccount:
         state.derivation && appRequest.account
-          ? state.derivation.address !== appRequest.account.freshAddress
+          ? state.derivation.address !== appRequest.account.freshAddress &&
+            state.derivation.address !== appRequest.account.seedIdentifier // Use-case added for Hedera
```

Comment:
@gre please review and confirm this is safe in your opinion :pray:

Reply:
afaik this is not safe. at first glance i think this is breaking all implementations as `address` is not `seedIdentifier`. I don't understand why the address != freshAddress is not enough to detect "it's wrong account for device". but what's sure is that seedIdentifier is never address in all our implementations

Reply:
what's bad is, that means freshAddressPath+freshAddress idea that both are coupled is broken and it might break in many part that expect this to match.

Reply:
Do you mean it can have an impact on existing coins? I doesn't seem possible because as we discussed no current integration have `address from device === seedIdentifier`.
If it can have an impact for Hedera only, then it can be addressed with their own integration, or we can consider a further step.
I manually tested that this part works for Hedera, and Bitcoin as an example of other coin.
I can test other parts but I don't know where it could break.

Reply:
it might be safe but it will require QA to be sure imo